File: gamefiles/gamedraw.py
import os
clear = lambda: os.system('cls')

def gamedraw(gamedata):
    ball = gamedata['ball']
    gameBoard = gamedata['room']     
    clear()    
    print(' ' + ('_' * gameBoard['width']))
    for y in range(gameBoard['height']):
        stringToPrint = '|'
        for x in range(gameBoard['width']):
            if (ball['x'] == (x + 1)) and (ball['y'] == (y+1)):
                stringToPrint = stringToPrint + '*'
            else:
                stringToPrint = stringToPrint + ' '
        print(stringToPrint + '|')
    print(' ' + ('_' * gameBoard['width']))

File: gamefiles/gamedraw.js

//Draw the gameboard and the ball using ASCII graphics (TEXT)
module.exports = (gameData) => {
    const ball = gameData.ball;
    const gameBoard = gameData.room;
    console.clear();    
    console.log(' ' + '_'.repeat(gameBoard.width));
    for(var y = 0; y < gameBoard.height; y++){
        var stringToPrint = '|'
        for(var x = 0; x < gameBoard.width; x++){
            if((ball.x === (x + 1) && (ball.y === (y+1))) ){
                stringToPrint+= '*';
            } else {
                stringToPrint+= ' ';
            }
        }        
        console.log(stringToPrint + '|');
    }    
    console.log(' ' + '_'.repeat(gameBoard.width));
}
